notebooks/image_models/solutions/conditional_gan.ipynb

```diff
@@ -0,0 +1,1190 @@
+{
```

Comment:
<ul><li>Why standard in the function name? Remove?</li><li>It feels the "create_genrator_block" is a bit overkill. It makes the structure of the generator harder to parse at a glance, and defining the create_generator_block takes as long as defining it in terms of lines of code. I suggest merging this cell and the one above and making the network structure very apparent with the code formatting (maybe remove the prefix <code>tf.keras.layers</code>).</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/asl-ml-immersion/pull/222/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Removed! Thanks